course/src/isometry.cc

```diff
@@ -0,0 +1,115 @@
+#include <isometry.h>
+
+namespace ekumen {
+  namespace math {
+
+  const Vector3 Vector3::kUnitX(1.,0,0);
+  const Vector3 Vector3::kUnitY(0,1.,0);
+  const Vector3 Vector3::kUnitZ(0,0,1.);
+  const Vector3 Vector3::kZero(0,0,0);
+
+  Vector3 Vector3::operator + (const Vector3 & obj) const
+  {
+    return Vector3( x_ + obj.x(), y_ + obj.y(),z_ +  obj.z());
+  }
+
+  Vector3 Vector3::operator - (const Vector3 & obj) const
+  {
+    return Vector3( x_ - obj.x(), y_ - obj.y(),z_ -  obj.z());
+  }
+
+  bool Vector3::operator == (const std::initializer_list<double> & obj) const
+  {
+    if (obj.size() != 3) return false;
+    int i = 0;
+    for (auto & e : obj)
+    {
+      if (*data_[i] != e) return false;
+      i++;
+    }
+    return true;
+  }
+
+  bool Vector3::operator == (const Vector3 & obj) const
+  {
+    if (obj.x() != x_ || obj.y() != y_ || obj.z() != z_) return false;
+    return true;
+  }
+
+  bool Vector3::operator != (const std::initializer_list<double> & obj) const
+  {
+    if (obj.size() != 3) return true;
```

Comment:
Fix format. 

Better throw an error. Having the length be an unexpected number is to the same as the vector being not equal.